web/client/components/TOC/fragments/WMSLegend.jsx

```diff
@@ -49,6 +68,14 @@ class WMSLegend extends React.Component {
     canShow = (node) => {
         return node.visibility || !this.props.showOnlyIfVisible;
     };
+
+    useLegendOptions = () =>{
```

Comment:
Why > 12? I it is a limit, I think that also the UI should be explicit on this limitation (e.g. input doesn't allow these values)